### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by TequilaSunset1998** [Slow-Scallion4183/Adventure-Colussus#36] [General]
Hi all, I recently got invited to contribute to this project through Reddit. I've never had any experience with GitHub and I've only started learning python last month. I think it's awesome what yall are doing here and I'm glad to be apart of it. Right now this is a learning experience for me so I probably can't contribute much right now, but I'm excited to contribute the more I start to learn the basics of git and python.

**Comment by ghost** [comment]
me too! welcome! the leader of this project himself is a beginner and is learning through coding in here.

**Comment by Slow-Scallion4183** [comment]
Sounds good mate! If you need anything be sure to start a discussion
